_posts/en/2016-02-26-return-values-with-the-new-operator.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+
+title: Return Values with the 'new' Operator
+tip-number: 50
+tip-username: Morklympious
+tip-username-profile: https://github.com/morklympious
+tip-tldr: Understand what gets returned when using new vs. not using new.
+
+categories:
+    - en
+---
+
+You're going to run into some instances where you'll be using `new` to allocate new objects in JavaScript. It's going to blow your mind unless you read this tip to understand what's happening behind the scenes.
+
+The `new` operator in JavaScript is an operator that, under reasonable circumstances, return a new instance of an object. Let's say we have a constructor function:
+
+````js
+function Thing() {
+  this.one = 1;
+  this.two = 2;
+}
+
+var myThing = new Thing();
+
+myThing.one // 1
+myThing.two // 2
+````
+
+__Note__: `this` refers to our newly created object when we use `new` to create it. Otherwise (if created without `new`,) `this` is going to point to the global object, which is `window`;
```

Comment:
Your current phrasing suggests that, without `new`, an object is still created. This is of course incorrect, so I suggest the following instead:

__Note__: `this` refers to the new object that was created by `new`. Otherwise, if `Thing()` is called without `new`, **no object is created**, and `this` in `Thing()` will point to the global object. This means that:

1. you'll suddenly have two new global variables named `one` and `two`, and
2. `myThing` is...nothing.

```js
var myThing = Thing();
myThing     // undefined
one       // 1
two       // 2
```